pkg/controller/backup_finalizer_controller.go

```diff
@@ -186,7 +191,16 @@ func (r *backupFinalizerReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	}
 	backup.Status.CompletionTimestamp = &metav1.Time{Time: r.clock.Now()}
 	recordBackupMetrics(log, backup, outBackupFile, r.metrics, true)
-
+	CSISnapshotsNotReady := errors.New("CSI snapshots not ready")
+	retry.OnError(retry.DefaultBackoff, func(err error) bool {
```

Comment:
The question here is how long do we wanna wait for CSI to complete.

At this phase it is also probably safe to unblock a new backup, not sure if that's the case already.